nsqadmin/static/js/models/node.js

```diff
@@ -0,0 +1,25 @@
+var $ = require('jquery');
+
+var AppState = require('../app_state');
+var Backbone = require('backbone');
+
+var Node = Backbone.Model.extend({
+    idAttribute: 'name',
+
+    constructor: function Node() {
+        Backbone.Model.prototype.constructor.apply(this, arguments);
+    },
+
+    url: function() {
+        return AppState.url('/nodes/' + this.get('name'));
+    },
+
+    tombstoneTopic: function(topic) {
+        return $.ajax(this.url(), {
+            'data': JSON.stringify({'topic': topic}),
+            'method': 'DELETE'
+        });
```

Comment:
I think this might also work?
```
this.destroy({
    data: {'topic': topic}
});
``` 

But is also less clear as to what it's actually doing (and might result in having this `node` removed from its collections?